### PR TITLE
Improve

### PR DESCRIPTION
**Pull request by qiankanglai**
Avoid calculate Color*Opacity every time
Add Mesh Name for Profiling
Improve Shader for lower bandwidth

### CODE REVIEW

File: Assets/Live2D/Cubism/Rendering/CubismRenderer.cs [hunk 1]
```diff
@@ -581,11 +581,11 @@ public void ApplyVertexColors()
         {
             var colors = VertexColors;
 
-
+            var col = Color;
```
Comment (itoh-at-live2d-com) [inline Assets/Live2D/Cubism/Rendering/CubismRenderer.cs]: Can you use meaningful name?
e.g. `var color = Color;`

File: Assets/Live2D/Cubism/Rendering/CubismRenderer.cs [hunk 2]
```diff
@@ -673,7 +673,7 @@ private void TryInitializeMesh()
                     uv = drawable.VertexUvs,
                     triangles = drawable.Indices
                 };
-
+                mesh.name = "Live2D";
```
Comment (itoh-at-live2d-com) [inline Assets/Live2D/Cubism/Rendering/CubismRenderer.cs]: You have to separate the commit.
Please make new commit for this modify.

**Comment by qiankanglai** [comment]
Commits have been reorganized 😄

File: Assets/Live2D/Cubism/Rendering/CubismRenderer.cs [hunk 3]
```diff
@@ -673,6 +673,7 @@ private void TryInitializeMesh()
                     uv = drawable.VertexUvs,
                     triangles = drawable.Indices
                 };
+                mesh.name = "Live2D";
```
Comment (itoh-at-live2d-com) [inline Assets/Live2D/Cubism/Rendering/CubismRenderer.cs]: We want to use drawable name for mesh name.
So, if you want to use this name, please use on your own branch.

Comment (qiankanglai) [reply Assets/Live2D/Cubism/Rendering/CubismRenderer.cs]: Drawable name could work too. I just want to avoid empty ones. I'll fix it

**Comment by itoh-at-live2d-com** [comment]
Thank you! :D

**Pull request merged**